database/migrations/2024_03_12_154408_remove_nests_table.php

```diff
@@ -28,10 +28,14 @@ public function up(): void
         }
 
         Schema::table('eggs', function (Blueprint $table) {
-            if (Schema::getConnection()->getDriverName() !== 'sqlite') {
-                $table->dropForeign('service_options_nest_id_foreign');
-            } else {
-                $table->dropForeign(['nest_id']);
+            switch (Schema::getConnection()->getDriverName()) {
+                case 'mariadb':
+                case 'mysql':
+                    $table->dropForeign('service_options_nest_id_foreign');
+                    break;
+                case 'sqlite':
+                    $table->dropForeign(['nest_id']);
+                    break;
```

Comment:
```suggestion
            $table->dropForeign(['nest_id']);
```
Those are old format